include/swift/Basic/Malloc.h

```diff
@@ -44,4 +44,4 @@ inline void AlignedFree(void *p) {
   
 }
 
-#endif
+#endif /* __SWIFT_MALLOC_H__ */
```

Comment:
While you are here, could you change the include guard to be `SWIFT_BASIC_MALLOC_H`?